Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2_test.py

```diff
@@ -132,22 +135,48 @@
     '    </soap:Body>' + \
     '</soap:Envelope>'
 
+GET_RESPONSE_NOT_SUCCESSFUL_JSON = {"IsSuccessful": False, "RequestedObject": None,
+                                    "ValidationMessages": [{"Reason": "Validation", "Severity": 3,
+                                                            "MessageKey": "ValidationMessageTemplates"
+                                                                          ":LoginNotValid",
+                                                            "Description": "",
+                                                            "Location": -1,
+                                                            "ErroredValue": None,
+                                                            "Validator": "ArcherApi."
+                                                                         "Controllers.Security"
+                                                                         "Controller, ArcherApi, "
+                                                                         "Version=6.5.200.1045, "
+                                                                         "Culture=neutral, "
+                                                                         "PublicKeyToken=null",
+                                                            "XmlData": None,
+                                                            "ResourcedMessage": None}]}
+
+GET_RESPONSE_SUCCESSFUL_JSON = {"IsSuccessful": True, "RequestedObject": {'SessionToken': 'session-id'}}
+
 
 def test_extract_from_xml():
     field_id = extract_from_xml(XML_FOR_TEST, 'Envelope.Body.GetValueListForField.fieldId')
     assert field_id == '6969'
 
 
-def test_get_level_by_app_id(requests_mock):
```

Comment:
please test the login in different method

Reply:
Done